source/includes/_endpoint-dataset.md

```diff
@@ -722,6 +722,36 @@ will be brought over to the current dataset:
 * Personal variables
 * Permissions
 
+The response will be a `shoji:entity` containing as a body an object with
+keys for each entity type that uas not been copied. In the case of variables
+these entities will display their name, alias and owner (if personal).
+
+All the URLs will refer to entities on the source dataset.
+
+```json
+{
+    "element": "shoji:entity",
+    "body": {
+        "variables": {
+            "http://app.crunch.io/dataset/:sourceid/variables/:varid/": {
+                "name": "Variable name",
+                "alias": "Variable alias",
+                "owner_url": "http://app.crunch.io/users/.../
```

Comment:
Fix your JSON. Make real-looking URLs. Use https. Don't use ellipses. Close the " on this line.